server/test/e2e/setup.ts

```diff
@@ -12,10 +8,10 @@ export default async () => {
     .withReuse()
     .start();
 
+  process.env.NODE_ENV = 'development';
+  process.env.TYPESENSE_ENABLED = 'false';
+  process.env.IMMICH_MACHINE_LEARNING_ENABLED = 'false';
+  process.env.IMMICH_TEST_ENV = 'true';
```

Comment:
I think this environment variable is really just determining how the `infra` module is built. Maybe we can have a more explicit test module that is available for import directly, rather than controller by this env.

Reply:
Thanks, I removed the env var and it seems to work fine without it